server/lib/flow-event.js

```diff
@@ -56,12 +57,20 @@ function isDNT (request) {
   return request.headers.dnt === '1';
 }
 
-function limitLength (param) {
-  if (param && param.length > MAX_PARAM_LENGTH) {
-    return param.substr(0, MAX_PARAM_LENGTH);
+function limitLength (data) {
+  if (data && data.length > MAX_DATA_LENGTH) {
+    return data.substr(0, MAX_DATA_LENGTH);
+  }
+
+  return data;
+}
+
+function sanitiseData (data) {
+  if (data === 'none') {
```

Comment:
Not sure where this default value comes from, presumably elsewhere in the content server. We need to ditch it anyway, for consistency with how the events are emitted in the auth server.